### ISSUE TITLE
Read and Write SDO index with error node

### ISSUE THREAD

**Issue by daddi1987** [CANopenNode/CANopenNode#534]
Hello to all of you. Without putting the board containing the STM32 into Emergency, I am unable to read or write.
I request that the master node read from my node to the index 3001 00. The request is completed, and the node answers with the result, but it returns an error.
My slave node is 15(F)
![image](https://github.com/user-attachments/assets/cda90b47-f0c0-4e49-9796-7f332c55d7e8)

**Comment by daddi1987** [comment]
The Error 8120 Passive node Error caused by absent to other node in the network. but not undestended first error 00 00 00 01. 
do you have any ideas?

**Comment by CANopenNode** [comment]
01 is a CO_EM_CAN_BUS_WARNING. Emergency is also sent on error reset with error code=0000.

Maybe just missing terminating resistor 120 ohm on the two ends?

**Comment by daddi1987** [comment]
Yes the both termination Is on , dont have a problem using the PDO. Different with the SDO command evry command put the bord in panic mode.

**Comment by CANopenNode** [comment]
SDO server is basic functionality and should work without special configuration. But it works in different thread than PDO...

If you started your application from scratch it may be complex to configure everything properly. I'm not familiar with STM32. Maybe you can first try with existing example, which should work. Then transfer it to your project. Or ask question there.
